contrib/minimize-test-profile.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2017 Richard Hughes <richard@hughsie.com>
+#
```

Comment:
I think in this header need to provide instruction on how this is intended to be used;

IE `fwupdtool security --force --json | tee ./contrib/minimize-test-profile.py > foo.json`

Another useful thing could be to add a capture option if nothing on stdin/arg1 that just calls `fwupdtool security --force --json` and does everything for you and spits out a file.

Reply:
JsonBuilder can't remove nodes, so it's not trivial to do in C. Maybe we add a fwupd_security_attr_to_json_full() with some flags? I do like the stdout pipe thing.

Reply:
I did the pipe thing. I think pushing this early feature into the C or CLI API is a bit early.